include/cocaine/api/isolate.hpp

```diff
@@ -86,13 +86,14 @@ struct category_traits<isolate_t> {
         get(context_t& context, const std::string& name, const dynamic_t& args) {
             ptr_type instance;
 
-            auto lock_ptr = instances.synchronize();
```

Comment:
This change is out of scope of this PR. Moreover I'm pretty sure I've already seen these changes in 99b34ca8786a181e1773ed1ddf65f5122a2a77a2